seismic_site/buildings/models.py

```diff
@@ -0,0 +1,94 @@
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+from import_export import resources
+
+
+class SectorChoice(Enum):
+    s1 = "Sector 1"
+    s2 = "Sector 2"
+    s3 = "Sector 3"
+    s4 = "Sector 4"
+    s5 = "Sector 5"
+    s6 = "Sector 6"
+
+
+BUILDING_STATUS_CHOICES = [(0, "Alege"), (1, "Acceptat"), (-1, "Respins")]
```

Comment:
Let's use English for these status choices too.

Reply:
I actually have no expertise for this. @tudoramariei

Reply:
I agree with @danniel here
One of the next steps early on for both [client](https://www.freecodecamp.org/news/setting-up-internationalization-in-react-from-start-to-finish-6cb94a7af725/) and [backend](https://docs.djangoproject.com/en/3.0/topics/i18n/) will be to [internationalise our texts](https://stackoverflow.com/questions/44535194/i18n-internationalization-and-localization-in-react-django)
Since we want our code to be open source and open for work for everyone, having everything in English is the only step in that direction

Reply:
Can we limit the scope of this PR and open a new issue for that?

Reply:
ah, that's for sure; internationalisation is another topic for another time, here's just a discussion of using English 😅